src/Broadway/EventStore/Management/EventStoreManagementInterface.php

```diff
@@ -0,0 +1,19 @@
+<?php
+
+/*
+ * This file is part of the broadway/broadway package.
+ *
+ * (c) Qandidate.com <opensource@qandidate.com>
+ *
+ * For the full copyright and license information, please view the LICENSE
+ * file that was distributed with this source code.
+ */
+
+namespace Broadway\EventStore\Management;
+
+use Broadway\EventStore\EventVisitorInterface;
+
+interface EventStoreManagementInterface
+{
+    public function visitEvents(EventVisitorInterface $eventVisitor, Criteria $criteria = null);
```

Comment:
I'd rather not allow `Criteria` to be `null`. People can explicitly pass an empty criteria object if they need it.

If you agree, let's also swap the arguments.